modules/lazyload/module.php

```diff
@@ -115,10 +115,11 @@ public function __construct() {
 		add_action( 'wp_footer', function() {
 			?>
 			<script type='text/javascript' defer>
-				document.addEventListener( 'DOMContentLoaded', function() {
-					const dataAttribute = 'data-e-bg-lazyload';
-					const lazyloadBackgrounds = document.querySelectorAll( `[${ dataAttribute }]:not(.lazyloaded)` );
-					const lazyloadBackgroundObserver = new IntersectionObserver( ( entries ) => {
+				const lazyloadRunObserver = () => {
+				const dataAttribute = 'data-e-bg-lazyload';
```

Comment:
Please add indentation to everything under the `lazyloadRunObserver` function.

Reply:
Done